docs/tutorial.rst

```diff
@@ -375,6 +375,8 @@ incompatible versions of packages already installed with conda.
     session.install("contexter", "--no-deps")
     session.install("-e", ".", "--no-deps")
 
+``"mamba"`` is also allowed as a choice, which will use/require 
+`mamba <https://github.com/mamba-org/mamba>`_ instead of conda.
```

Comment:
```suggestion
``"mamba"`` is also allowed as a choice for ``venv_backend``, which will
use/require `mamba <https://github.com/mamba-org/mamba>`_ instead of conda.

```

We should mention `venv_backend` as it's not clear from the immediate context. The code example with `venv_backend="conda"` is quite a bit further up.